js/freewheel.js

```diff
@@ -460,9 +460,9 @@ OO.Ads.manager(function(_, $) {
             slotEndedCallbacks[ad.ad.getCustomId()] = _.bind(function(adId) {
                 amc.notifyPodEnded(adId);
               }, this, ad.id);
-            adStartedCallbacks[ad.ad.getCustomId()] = _.bind(function(details) {
-                amc.notifyLinearAdStarted(this.name, details);
-              }, this);
+            adStartedCallbacks[ad.ad.getCustomId()] = _.bind(function(adId, details) {
```

Comment:
is this new parameter already being used?  i don't see any other changes such as to the function calls